src/bokeh/models/widgets/tables.py

```diff
@@ -819,7 +819,7 @@ def __init__(self, *args, **kwargs) -> None:
 
     selectable = Either(Bool(True), Enum("checkbox"), help="""
     Whether a table's rows can be selected or not. Using ``checkbox`` is
-    equivalent  to ``True``, but makes selection visible through a checkbox
+    equivalent to ``True``, but makes selection visible through a checkbox
```

Comment:
I feel like we have informally settled on just using `True` rather than ` ``True`` ` in the last few years. I am not sure what the extra quoting really adds except to make it more cluttered in the ascii docstring version. As long as this is getting updated I'd lightly suggest making that change as well